web/book/src/language-features/standard-library/reading-files.md

```diff
@@ -0,0 +1,18 @@
+# Reading files
+
+We have a couple of functions named `read_*`, which ask the DB to read files,
+designed for DuckDB:
+
+```prql
+from (read_parquet 'artists.parquet')
+join (read_csv 'albums.csv') [==track_id]
+```
```

Comment:
What do you think about not even having `from`?

```suggestion
```prql
read_parquet 'artists.parquet'
join (read_csv 'albums.csv') [==track_id]
```
```

Reply:
But then we can't use with `join` etc?

(the very first version of PRQL, for like 3 days, didn't even have a `from`, it was just `employees | select`...)